WORKSPACE

```diff
@@ -10,6 +10,13 @@ git_repository(
     remote = "https://github.com/vmax/bazel-deps-deploy",
 )
 
+# deployed version of //tools:simple_formatter
```

Comment:
@oferb 
>Are we using this?

Yes, [here](https://github.com/google/startup-os/blob/6f1aecf43ecfe4fc865682975cbd0ddbbc013683/check-formatting.sh#L21)

>If so, why not use only //tools:simple_formatter?

Then it would take time to build on `lint` stage, but it's way too much since it depends on `protobuf`